src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Figure[] FIGURES = new Figure[] {
+            new Circle(new Random().nextInt(100), ColorSupplier.randomColor()),
```

Comment:
Take 100 as a constant

Reply:
If I will take 100 as a constant Circle figure will not have random parameter.